.travis.yml

```diff
@@ -43,7 +43,7 @@ before_install:
 install:
 - mvn -version -B
 script:
-- mvn verify -Dwildfly.logging.console.level=DEBUG -Djboss-as.logging.console.level=DEBUG -B
+- mvn verify -Dwildfly.logging.console.level=INFO -Djboss-as.logging.console.level=INFO -B
```

Comment:
Out of curiosity, was there a problem on Travis? Or is there another reason for change?